### PR TITLE
[DE] Add History of osu!, History of the osu! wiki, and Grade; update `.osb` and `.osz` articles

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Among adding articles, this PR fixes inconsistencies in the `osz` and `osb` file formats articles with their respective English version. Importantly, these two articles should have been marked as outdated.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/File_formats/Osz_(file_format)/de.md [hunk 1]
```diff
@@ -1,9 +1,3 @@
 # .osz (Dateiformat)
 
-**.osz** ist ein Beatmapdateiformat, welches Hitsounds, .osu Datei, Skin Elemente, .osb Datei und Storyboard Elemente enthalten kann.
-
-Um eine .osz Datei zu erstellen, öffnet man einfach die Beatmap, die man als .osz exportieren möchte, im Editor und klickt oben-links auf "Datei" und dann auf "Paket exportieren", siehe Bild.
-
-![Exporting .osz](img/DE-.osz_export.jpg "Exporting .osz")
-
-Die exportierte Datei wird dann im osu! Verzeichnis im "Export" Ordner gespeichert und aufbewahrt.
+**.osz** ist ein gesamtheitliches Dateiarchiv, welches Beatmaps und alle zusätzlichen Ressourcen wie Skins oder Hitsounds enthält.
```
Comment (The-Last-Cookie) [inline wiki/Client/File_formats/Osz_(file_format)/de.md:3]: Not sure if this must **explicitely** be just one skin (`wie ein Skin`) or several skins (see current version).

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Client/File_formats/Osz_(file_format)/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Client/File_formats/Osz_(file_format)/de.md:3]: `Skinelemente` might be a possibility too 🤔 

You just need to place an image with the correct name in the `.osz` file, and the game uses your custom image when playing the beatmap.

File: wiki/Gameplay/Grade/de.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - Benotung
+---
+
+# Note
+
+*Für andere Bedeutungen, siehe [Rang (Begriffsabgrenzung)](/wiki/Disambiguation/Rank).*
+
+**Noten** sind eine Reihe an Buchstaben, die die Leistung des Spielers auf einer bestimmten [Beatmap](/wiki/Beatmap) repräsentieren. Sie begleiten die [Gesamtpunktzahl](/wiki/Gameplay/Score) auf der Ergebnisanzeige und tauchen in allen Listen an der Seite von Scores auf.
+
+Von der niedrigsten zur höchsten sind die etwaigen Noten D, C, B, A, S und SS. Silver S und SS Noten sind nur erreichbar, wenn die Spielmodifikationen [Hidden](/wiki/Game_modifier/Hidden), [Flashlight](/wiki/Game_modifier/Flashlight) oder [Fade In](/wiki/Game_modifier/Fade_In) verwendet werden.
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Gameplay/Grade/de.md:18]: TIL Silver S is translated as `Silber-S`, so it technically needs to `Silber-S- und SS-Noten`. As this looks a bit bogus (too many dashes), I think that `Silber S und SS Noten` is fine too.

File: wiki/History_of_osu!/de.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- [2007](/wiki/History_of_osu!/2007) - Anfängliche Spielentwicklung
+- [2008](/wiki/History_of_osu!/2008) - Taiko und Catch the Beat, mehr Mods und wesentliche Verbesserungen
+- [2009](/wiki/History_of_osu!/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/History_of_osu!/2011) - osu!-Wiki
+- [2012](/wiki/History_of_osu!/2012) - Fünf Jahre osu!, osu!mania, Breitbildunterstützung und visuelle Einstellungen
+- [2013](/wiki/History_of_osu!/2013) - Überarbeitung der UI, HD-Skins, verbesserte Breitbildunterstützung, osu!mania-Editor, Sammlungen im Spiel, moddingV2 wurde vorgestellt, Disqus, osu!api, Overlays für Genauigkeit, Comic Fiesta 2013, unbegrenzte Downloads und osu!cuttingedge (nur osu!supporters)
+- [2014](/wiki/History_of_osu!/2014) - BPM-Scrolling, Intro-/Outro-Sequenzen im Spiel, osu!cuttingedge (für alle) und Schnee!
+- [2015](/wiki/History_of_osu!/2015) - jizz (gestaltete osu!web neu), osu!weekly und osu!coins
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/History_of_osu!/de.md:18]: TIL jizz was the domain name and not one of a user, as originally thought.

File: wiki/Client/File_formats/Osz_(file_format)/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Client/File_formats/Osz_(file_format)/de.md:3]: yeah, `Skinelemente` sounds best :+1:

File: wiki/Gameplay/Grade/de.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Gameplay/Grade/de.md:18]:
```suggestion
Von der niedrigsten zur höchsten sind die etwaigen Noten D, C, B, A, S und SS. Silber S und SS Noten sind nur erreichbar, wenn die Spielmodifikationen [Hidden](/wiki/Game_modifier/Hidden), [Flashlight](/wiki/Game_modifier/Flashlight) oder [Fade In](/wiki/Game_modifier/Fade_In) verwendet werden.
```

**Review by respektive — DISMISSED** [review]

**Comment by cl8n** [comment]
`needs_cleanup` and other comments only relevant to the English version of the article don't need to be copied over

also, try to keep PRs focused on one topic at a time

**Review by cl8n — APPROVED** [review]

**Pull request merged**